src/dash.ts

```diff
@@ -33,4 +35,33 @@ export class Dash {
 
         return '';
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns the command to handle the URI in the command line.
+     *
+     * @return {string} name and flags of the command that takes the URI as an argument
+     */
+    getDashURIHandler(): String {
+        // hacky switch statement
+        return {
+            'darwin': 'open -g',
+            'linux': 'zeal',
+            // start Zeal before we pass it a URI so it works whether Zeal is running in background or not.
+            // Why is this needed? I have no idea, but I think Qt has to start up a listener on cmd before it will work.
+            'win32': 'start dash-plugin:// && start'
+        }[platform()] || 'zeal';
```

Comment:
looks good @szhongren 👍  

seems like we can omit the string in the key so it will be

```js
return {
  darwin: 'open -g',
  linux: 'zeal',
  ...
}
```
it is a small matter though